openshift/postgresql.dc.json

```diff
@@ -59,12 +59,24 @@
             "value": "1Gi"
         },
         {
-            "description": "Version of PostgreSQL image to be used (9.2, 9.4, 9.5 or latest).",
+            "description": "Version of PostgreSQL image to be used (9.5 or 9.6).",
             "displayName": "Version of PostgreSQL Image",
             "name": "IMAGE_STREAM_VERSION",
             "required": true,
-            "value": "9.5"
+            "value": "v1-stable"
         },
+        {
+            "description": "Indicator to enable pgcrypto extension (provided out-of-the-box with PostgreSQL).",
+            "displayName": "Flag to enable pgcrypto PostgreSQL extension.",
+            "name": "PGCRYPTO_EXTENSION",
+            "value": "Y"
+        },
+        {
+            "description": "Indicator to enable postgis extension (from official PostgeSQL YUM repo).",
+            "displayName": "Flag to enable postgis PostgreSQL extension.",
+            "name": "POSTGIS_EXTENSION",
+            "value": "Y"
```

Comment:
will we need to recreate the database or can we add a migration `create extension postgis` and re-use the existing data pvc?

Reply:
We cannot update 9.5 -> 9.6 (see https://devopspathfinder.slack.com/archives/C0PJD4LBU/p1544130149022500?thread_ts=1544127638.018600&cid=C0PJD4LBU).  So yes we'll need to:
- bring up Caddy to intercept users
- scale down application server
- pg_dump (using @DerekRoberts 's fancy https://github.com/bcgov/gwells/blob/release/1.47.0/openshift/scripts/oc-dump.sh )
- merge https://github.com/bcgov/gwells/pull/1062 into Release branch
- deploy to DEV (which will work as it's a brand new DB each time)
- deploy TEST/STAGING (which will fail as it's persistent DB with v9.5)
- oc debug into TEST DB pod and `rm /userdata`
- let it deploy again into blank `/userdata` folder, upgrading to v9.6 with postgis 2.4
- pg_restore (using @DerekRoberts 's fancy https://github.com/bcgov/gwells/blob/release/1.47.0/openshift/scripts/oc-restore.sh)
- scale up application server
- disable Caddy 
- test on STAGING/TEST before trying on PROD